plugins/api-docs/report-alpha.api.md

```diff
@@ -13,12 +13,13 @@ import { EntityPredicate } from '@backstage/plugin-catalog-react/alpha';
 import { ExtensionBlueprintParams } from '@backstage/frontend-plugin-api';
 import { ExtensionDataRef } from '@backstage/frontend-plugin-api';
 import { ExtensionInput } from '@backstage/frontend-plugin-api';
-import { ExternalRouteRef } from '@backstage/frontend-plugin-api';
+import { ExternalRouteRef } from '@backstage/core-plugin-api';
```

Comment:
Right, so because the plugin still is old-system compatible you can now see visibly on the feature that the ref is old, which wasn't the case before. Is that feature field/type not consumed directly by any piece of code that could now get confused by that fact?

I gess the alternative would be to force-convert the refs still behind the scenes.